install/README.md

```diff
@@ -18,85 +18,82 @@ Once the CSM installation has completed, other product streams for the HPE Cray
 
 ## Topics
 
-1. [Preparing for a Re-installation](#preparing-for-a-re-installation)
-1. [Pre-Installation](#pre-installation)
-   1. [Boot Installation Environment](#1-boot-installation-environment)
-   1. [Import CSM Tarball](#2-import-csm-tarball)
-   1. [Create System Configuration](#3-create-system-configuration)
-   1. [Configure Management Network Switches](#4-configure-management-network-switches)
-   1. [Identify Inventory](#5-identify-inventory)
+1. [Preparing for a re-installation](#preparing-for-a-re-installation)
+1. [Pre-installation](#pre-installation)
+   1. [Boot installation environment](#1-boot-installation-environment)
+   2. [Import CSM tarball](#2-import-csm-tarball)
+   3. [Create system configuration](#3-create-system-configuration)
+   4. [Configure management network switches](#4-configure-management-network-switches)
+   5. [Identify inventory](#5-identify-inventory)
 1. [Installation](#installation)
-    1. [Deploy Management Nodes](#1-deploy-management-nodes)
-    1. [Install CSM Services](#2-install-csm-services)
-    1. [Validate CSM Health Before Final NCN Deployment](#3-validate-csm-health-before-final-ncn-deployment)
-    1. [Deploy Final NCN](#4-deploy-final-ncn)
-    1. [Configure Administrative Access](#5-configure-administrative-access)
-    1. [Validate CSM Health](#6-validate-csm-health)
-    1. [Configure Prometheus Alert Notifications](#7-configure-prometheus-alert-notifications)
-    1. [Update Firmware with FAS](#8-update-firmware-with-fas)
-    1. [Prepare Compute Nodes](#9-prepare-compute-nodes)
-1. [Next Topic](#next-topic)
-1. [Troubleshooting Installation Problems](#troubleshooting-installation-problems)
+   1. [Deploy management nodes](#1-deploy-management-nodes)
+   2. [Install CSM services](#2-install-csm-services)
+   3. [Validate CSM health](#3-validate-csm-health)
+   4. [Deploy final NCN](#4-deploy-final-ncn)
+   5. [Configure administrative access](#5-configure-administrative-access)
+   6. [Validate CSM health](#6-validate-csm-health)
+   7. [Configure Prometheus alert notifications](#7-configure-prometheus-alert-notifications)
+   8. [Update firmware with FAS](#8-update-firmware-with-fas)
+   9. [Prepare compute nodes](#9-prepare-compute-nodes)
+1. [Next topic](#next-topic)
+1. [Troubleshooting installation problems](#troubleshooting-installation-problems)
 
 The topics in this chapter need to be done as part of an ordered procedure so are shown here with numbered topics.
 
-> **`NOTE`** If problems are encountered during the installation, some topics have their own [troubleshooting sections found in the operations index](../operations/README.md)
```

Comment:
Why are these changing?

Reply:
I don't know the answer to that one -- I don't think I took that line out.

Oh, wait, I think you're asking why remove the backticks from around the word NOTE? Mainly because I don't think backticks are intended to be used as syntax highlighting similar to bolding or italics. Backticks have special effects associated with them (for example, that you assume the contents should be shielded from things like spell checks). I believe the intention is for them to be used to contain special strings that aren't just regular text. So I don't think it's appropriate to use them just for emphasis (particularly when there are other mechanisms available to do that).